keyboards/splitkb/kyria/keymaps/squigglybob/keymap.c

```diff
@@ -0,0 +1,315 @@
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+enum layers {
+    _DVERTY = 0,
+    _QWERTY,
+    _DVORAK,
+    _NAV,
+    _SYM,
+    _SYM_DVERTY,
+    _FUNCTION,
+    _ADJUST,
+};
+
+// Aliases for readability
+#define QWERTY DF(_QWERTY)
+#define DVERTY DF(_DVERTY)
+#define DVORAK DF(_DVORAK)
+
+#define SYM MO(_SYM)
+#define SYM_DVERTY MO(_SYM_DVERTY)
+#define NAV MO(_NAV)
+#define FKEYS MO(_FUNCTION)
+#define ADJUST MO(_ADJUST)
+
+#define CTL_ESC MT(MOD_LCTL, KC_ESC)
+#define CTL_QUOT MT(MOD_RCTL, KC_QUOTE)
+#define CTL_MINS MT(MOD_RCTL, KC_MINUS)
+#define ALT_ENT MT(MOD_LALT, KC_ENT)
+#define CTL_ENT MT(MOD_LCTL, KC_ENT)
+#define SFT_SPC MT(MOD_LSFT, KC_SPC)
+#define ALT KC_LEFT_ALT
+
+#define UK_BKSL KC_NUBS     // UK backslash
+#define UK_PIPE S(UK_BKSL)  // UK pipe symbol
+#define DV_AT S(KC_Q)       // Dvorak @ symbol
+
+// Note: LAlt/Enter (ALT_ENT) is not the same thing as the keyboard shortcut Alt+Enter.
+// The notation `mod/tap` denotes a key that activates the modifier `mod` when held down, and
+// produces the key `tap` when tapped (i.e. pressed and released)
+
+// clang-format off
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+/*
+ * Base Layer: QWERTY (default map)
+ *
+ * ,-------------------------------------------.                              ,-------------------------------------------.
+ * |  Tab   |   Q  |   W  |   E  |   R  |   T  |                              |   Y  |   U  |   I  |   O  |   P  |  Bksp  |
+ * |--------+------+------+------+------+------|                              |------+------+------+------+------+--------|
+ * |Ctrl/Esc|   A  |   S  |   D  |   F  |   G  |                              |   H  |   J  |   K  |   L  | ;  : |Ctrl/' "|
+ * |--------+------+------+------+------+------+-------------.  ,-------------+------+------+------+------+------+--------|
+ * | LShift |   Z  |   X  |   C  |   V  |   B  | [ {  |CapsLk|  |F-keys|  ] } |   N  |   M  | ,  < | . >  | /  ? | RShift |
+ * `----------------------+------+------+------+------+------|  |------+------+------+------+------+----------------------'
+ *                        |Adjust| LGUI | LAlt/| Space| Nav  |  | Sym  | Space| AltGr| RGUI | Menu |
+ *                        |      |      | Enter|      |      |  |      |      |      |      |      |
+ *                        `----------------------------------'  `----------------------------------'
+ */
+    [_QWERTY] = LAYOUT(
```

Comment:
Just a heads up, order here doesn't matter.  Layers will use the order used in the enum.  So this will be  layer 1, dverty will be layer  0, etc.

Reply:
That makes sense.
So nothing needs to be done here? More of an info review?

Reply:
Mostly a double check, just in case.   No action needed.